src/main/java/sleeper/main/MessageFormatting.java

```diff
@@ -0,0 +1,47 @@
+package sleeper.main;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import org.bukkit.ChatColor;
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.util.Set;
+
+public class MessageFormatting {
+    
+    Main plugin;
+    
+    public MessageFormatting(Main plugin) {
+        this.plugin = plugin;
+    }
+    
+    Set<String> allowedTypes = Set.of("MINECRAFT", "MINIMESSAGE");
+    
+    // Setting value
+    String formattingType = "MINIMESSAGE";
```

Comment:
As this also works as the default for faulty configs I think it would be best to have it as the normal minecraft style
```suggestion
    String formattingType = "MINECRAFT";
```

Reply:
Done